pom.xml

```diff
@@ -4,9 +4,4 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
```

Comment:
pom.xml is missing dependencies for junit and selenium as well as fix for java level 5 issue
Please add missing sections basing on example from https://docs.google.com/presentation/d/1wbmTDlSZeV0kur9sCBpF5gzI9Z6qIWC2/edit#slide=id.p31

Reply:
@KatyaKulikova ^^